docs/en/operations/caches.md

```diff
@@ -25,5 +25,8 @@ Additional cache types:
 - Schema inference cache.
 - [Filesystem cache](storing-data.md) over S3, Azure, Local and other disks.
 - [Query cache](query-cache.md).
+- Format schema cache.
 
 To drop one of the caches, use [SYSTEM DROP ... CACHE](../sql-reference/statements/system.md#drop-mark-cache) statements.
+
+To drop the format schema cache, use the [SYSTEM DROP FORMAT SCHEMA CACHE](../sql-reference/statements/system.md#drop-format-schema-cache-system-drop-schema-format) statement.
```

Comment:
@bharatnc @randomizedcoder Two things are wrong in this PR:

- L. 30 should link to `../sql-reference/statements/system.md` (not: `../sql-reference/statements/system.md#drop-mark-cache`)
- L. 32 is clearly a special case of l. 30. We can delete l. 32.